west.yml

```diff
@@ -96,7 +96,7 @@ manifest:
       path: bootloader/mcuboot
     - name: mcumgr
       repo-path: sdk-mcumgr
-      revision: v0.1.0-ncs1
+      revision: f28a637db12c4b12fb2b18bddc6b2a0deaa95251
```

Comment:
```suggestion
      revision: pull/12/head
```

Reply:
@ioannisg Sorry for committing the suggestion to your changes (the github interface is very ambigius for me). Pleas squash them to your commit.